### ISSUE TITLE
Cannot navigate to instance details in REX

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#4025]
* Bring up coco pharmaceuticals tutorials environment - with config, start, building asset catalog notebooks run
* Navigate to REX UI
* Select cocoMDS2
* search for 'csv' (which should find 3 assets) 
* select all
* Assets shown on canvas
* Click on one for more detail.

Popup appears
```
The selected object is not local to the server being used and the enterprise option is not checked. You could retrieve it either by setting server to cocoMDS1 or by enabling the enterprise option
```

At this point I tried clicking on cocoMDS1 then clicking again on the asset - but I got the same warning.
I then repeated the search with cocoMDS1 selected, but this did not add existing nodes to the canvas, reporting:
```
No additional objects were returned in the traversal
```
I then tried 'clear' and repeated the search. This cleared the canvas. At this point 'cocoMDS1 was selected
Assets were added to diagram, but upon clicking - exactly the same error message

To clarify - selecting Enterprise does work fine as expected.

**Comment by grahamwallis** [comment]
Do you know which repository is the home for the instances that were retrieved?

**Comment by planetf1** [comment]
cocoMDS1

**Comment by grahamwallis** [comment]
Fixed by #4080
